docs/sources/administration/plugin-management/index.md

```diff
@@ -162,21 +152,30 @@ plugins:
   - redis-datasource
 ```
 
-### Install plugin on local Grafana
+When the update is complete, you'll see a confirmation message that the installation was successful.
```

Comment:
When the update is complete, a confirmation message will indicate it was successful.

Reply:
thanks, adjusted